linked_list/cpp/test.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include "LinkedList.hpp"
+
+using namespace std;
+
+int main() {
+  LinkedList<int> intList;
+
+  intList.insertTail(2);
+  intList.insertTail(4);
+  intList.insertHead(1);
+  intList.insertAfter(2, 3);
+
+  // Print out each element sequentially using an iterator
+  int size = intList.size();
+  if (size == 0) {
+      cout << "Empty";
+  } else {
+      for (auto iter = intList.begin(); iter != intList.end(); ++iter) {
```

Comment:
Line contains following spacing inconsistencies:
- Spaces used instead of tabs.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpqyw85au0/linked_list/cpp/test.cpp
+++ b/tmp/tmpqyw85au0/linked_list/cpp/test.cpp
@@ -16,7 +16,7 @@
   if (size == 0) {
       cout << "Empty";
   } else {
-      for (auto iter = intList.begin(); iter != intList.end(); ++iter) {
+	  for (auto iter = intList.begin(); iter != intList.end(); ++iter) {
           cout << *iter << " ";
       }
   }
```